### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/advanced/generate-clients.md`

### PR DESCRIPTION
**Pull request by funny-cat-happy**
add translation for generate-client

### CODE REVIEW

**Review by eVery1337 — APPROVED** [review]

**Comment by ChoyeonChern** [comment]
Some of the content seems to have been over-translated, I personally think the original documentation is easier to read, this is the first batch of translations, consider talking to the original contributor

**Comment by ChoyeonChern** [comment]
提交似乎不符合规范，没有编辑入口文件这是一个新翻译要提交路由的

**Comment by funny-cat-happy** [comment]
> 提交似乎不符合规范，没有编辑入口文件这是一个新翻译要提交路由的

Do you mean add file router to mkdocs.yml like https://github.com/tiangolo/fastapi/pull/2261? But it is inherted from mkdocs of en.

File: docs/zh/docs/advanced/generate-clients.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+<img src="/img/tutorial/generate-clients/image02.png">
+
+您还将自动补全要发送的数据：
+
+<img src="/img/tutorial/generate-clients/image03.png">
+
+!!! tip
+    请注意 `Item` 模型中FastAPI应用程序中定义的 `name `和 `price` 的自动补全。
+
+您也将拥有发送的数据出现的内部错误：
```
Comment (xzmeng) [inline docs/zh/docs/advanced/generate-clients.md:117]:
```suggestion
如果发送的数据字段不符，你也会看到编辑器的错误提示:
```

File: docs/zh/docs/advanced/generate-clients.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+<img src="/img/tutorial/generate-clients/image04.png">
+
+回应的对象也拥有自动补全:
```
Comment (xzmeng) [inline docs/zh/docs/advanced/generate-clients.md:121]:
```suggestion
响应(response)对象也拥有自动补全:
```

File: docs/zh/docs/advanced/generate-clients.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+
+### 客户端方法名称
+
+现在生成的方法名像 `createItemItemsPost` 看起来不太简洁:
+
+```TypeScript
+ItemsService.createItemItemsPost({name: "Plumbus", price: 5})
+```
+
+...这是因为客户端生成器为每个*路径操作*使用OpenAPI的内部 **操作 ID**。
```
Comment (xzmeng) [inline docs/zh/docs/advanced/generate-clients.md:164]:
```suggestion
...这是因为客户端生成器为每个 *路径操作* 使用OpenAPI的内部 **操作 ID(operation ID)**。
```

File: docs/zh/docs/advanced/generate-clients.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+例如，您可以确保每个*路径操作*都有一个标签，然后根据**标签**和*路径操作***名称**（函数名）来生成操作ID。
+
+### 自定义生成唯一ID函数
+
+FastAPI为每个*路径操作*使用一个**唯一ID**，它用于操作ID，也用于任何所需自定义模型的名称，用于请求或响应。
+
+你可以自定义该函数。它接受一个 `APIRoute` 对象作为输入，并输出一个字符串。
+
+例如，以下是一个示例，它使用第一个标签（你可能只有一个标签）和路径操作名称（函数名）。
```
Comment (xzmeng) [inline docs/zh/docs/advanced/generate-clients.md:184]:
```suggestion
例如，以下是一个示例，它使用第一个标签（你可能只有一个标签）和*路径操作*名称（函数名）。
```

File: docs/zh/docs/advanced/generate-clients.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+  "author": "",
+  "license": "",
+  "devDependencies": {
+    "openapi-typescript-codegen": "^0.20.1",
+    "typescript": "^4.6.2"
+  }
+}
+```
+
+生成新的客户端之后，你现在将拥有**清晰的方法名称**，具备**自动补全**、**内部错误**等功能：
```
Comment (xzmeng) [inline docs/zh/docs/advanced/generate-clients.md:248]:
```suggestion
生成新的客户端之后，你现在将拥有**清晰的方法名称**，具备**自动补全**、**错误提示**等功能：
```

**Review by xzmeng — COMMENTED** [review]
Excellent translation 👍 

A few suggestions on readability. Please let me know if I understand anything wrong.

BTW 建议文中“你”和“您”的使用统一一下，我个人比较喜欢“你”，“您”总是给我一种客气的距离感:)

**Comment by tiangolo** [comment]
📝 Docs preview for commit 7eabb8445c963151025fd59e389d3c4b9c1dd2c2 at: https://248f88bc.fastapitiangolo.pages.dev

**Review by xzmeng — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 505959290d07942e1f9defb8d5aa027dc2592e06 at: https://eb7407d5.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
Awesome, thanks @funny-cat-happy! 🚀 

And thanks for the reviews @xzmeng and @eVery1337 🙇

**Pull request merged**